pad_type: ps4
num_of_buttons: 13
axis_linear_x: 1
axis_linear_y: 0
axis_angular: 3
axis_linear_z: 5
axis_elevator: 10
scale_linear: 1.5
scale_angular: 10
scale_linear_z: 20.0
button_speed_up: 2
button_speed_down: 0
button_dead_man: 5
button_output_1: 2
button_output_2: 0
output_1: 10
output_2: 11
button_ptz_tilt_up: 7
button_ptz_tilt_down: 7
button_ptz_pan_right: 6
button_ptz_pan_left: 6
button_ptz_zoom_wide: 9
button_ptz_zoom_tele: 8
button_home: 6
tilt_increment: 5
pan_increment: 5
zoom_increment: 300
button_kinematic_mode: 7
joystick_dead_zone: "true" #useful for x-wam
cmd_service_home: rb1_base_controller/home
check_message_timeout: true
elevator_service_name: robotnik_base_control/set_elevator
cmd_topic_vel: /cmd_vel_mux/input/teleop
#cmd_topic_vel: teleop_velocity_smoother/raw_cmd_vel
